domenetjenester/brevbestiller/src/main/java/no/nav/foreldrepenger/melding/datamapper/domene/BeregningsresultatMapper.java

```diff
@@ -121,7 +121,10 @@ private static PeriodeType mapEnkelPeriode(BeregningsresultatPeriode beregningsr
         periode.setAntallTapteDager(BigInteger.valueOf(mapAntallTapteDagerFra(uttakResultatPeriode.getAktiviteter())));
         periode.setInnvilget(uttakResultatPeriode.isInnvilget() && !erGraderingAvslått(uttakResultatPeriode));
         PeriodeResultatÅrsak periodeResultatÅrsak = utledÅrsakskode(uttakResultatPeriode);
-        periode.setPeriodeFom(XmlUtil.finnDatoVerdiAvUtenTidSone(beregningsresultatPeriode.getBeregningsresultatPeriodeFom()));
+        if(uttakResultatPeriode.getFom().isBefore(beregningsresultatPeriode.getBeregningsresultatPeriodeFom())&& periodeResultatÅrsak.getKode().equals(PeriodeResultatÅrsak.HULL_MELLOM_FORELDRENES_PERIODER.getKode()) )
```

Comment:
Selv om det er mulig at if else ikke trenger { } så er det lurt å legge inn. Vi har hatt det som en del av kodestandarden i prosjektet tidligere i hvertfall. 
if (...) { 
kode;
} else {
kode2;
}

i steden for
if (...) kode;
else kode2;